routes/campgrounds.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+const router = express.Router();
+const Campground = require("../models/campground");
+const Comment = require("../models/comment");
+const middleware = require("../middleware");
+
+router.get("/campgrounds", function (req, res) {
+  //get all campground from
+  Campground.find({}, function (err, allCampgrounds) {
+    if (err) {
+      console.log(err);
+    } else {
+      res.render("campgrounds/index", { campgrounds: allCampgrounds });
+    }
+  });
+});
+
+// Create
+router.post("/campgrounds", middleware.isLoggedIn, function (req, res) {
+  // get data from form
+  var name = req.body.name;
```

Comment:
Instead of repeatedly using req.body.<param>, you can store it to a single variable and use that instead. For example:

const requestData = req.body.

Or it’s better to get all the data from the request and merge it together into single variable. For example:

const requestData = _.merge({}, req.body, req.query);

Something like that.